src/zjs_aio.c

```diff
@@ -19,7 +19,7 @@ static struct nano_sem aio_sem;
 
 typedef struct aio_handle {
     jerry_value_t pin_obj;
-    int32_t callback_id;
+    int16_t callback_id;
```

Comment:
Are you sure you save anything here? From jerryscript:
~~~
typedef uint32_t jerry_size_t;
~~~